jupyter-book/cellular_structure/integration.ipynb

```diff
@@ -812,9 +884,10 @@
    "source": [
     "> **How many genes to use?**\n",
     ">\n",
-    "> This is a question which doesn't have a clear answer. The authors of the **scvi-tools** package which we use below recommend between 1000 and 10000 genes but how many depends on the context including the complexity of the dataset and the number of batches.\n",
-    "> In general it is better to select slightly too many genes than to select too few and risk removing genes which are important for a rare cell type.\n",
-    "> However, more genes will also increase the time required to run the integration methods."
+    "> This is a question which doesn't have a clear answer. The authors of the **scvi-tools** package which we use below recommend between 1000 and 10000 genes but how many depends on the context including the complexity of the dataset and the number of batches. \n",
+    "> A survey from a previous best-practices paper {cite}`Lücken2019` indicated people typically use between 1000 and 6000 HVGs in a standard analysis.\n",
+    "> While selecting fewer genes can aid in the removal of batch effects {cite}`Lücken2022` (the most highly-variable genes often describe only dominant biological variation, we recommend selecting slightly too many genes than to select too few and risk removing genes which are important for a rare cell type or a pathway of interest.\n",
```

Comment:
Missing a closing bracket in this line

Reply:
closed via [ced74ec](https://github.com/theislab/single-cell-best-practices/pull/87/commits/ced74ec1d4314711ae2f5d16b0bc6d35eb6eaaa7)